example/src/Views/TopNavigationBar.tsx

```diff
@@ -1,6 +1,6 @@
 import React from 'react';
 import { StyleSheet, ScrollView, View, Alert } from 'react-native';
-import { Text, NavigationButton, Checkbox, TopNavigationBar, getColor, RadioButton, LinkProps, TopNavigationBarProps, Search, ContentSwitcher, styleReferenceBreaker } from '@carbon/react-native';
+import { Text, NavigationButton, Checkbox, TopNavigationBar, getColor, RadioButton, LinkProps, TopNavigationBarProps, Search, ContentSwitcher } from '@carbon/react-native';
```

Comment:
These ended up not being needed in many places ... concatenation is done by react-native and mutation guarding doesn't make sense when it's just consumed by the style prop